scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/quartz/AppScalingScheduleJob.java

```diff
@@ -0,0 +1,40 @@
+package org.cloudfoundry.autoscaler.scheduler.quartz;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.quartz.JobDataMap;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.quartz.JobKey;
+import org.springframework.scheduling.quartz.QuartzJobBean;
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+/**
+ * @author Fujitsu
+ *
+ */
+@Component
+public class AppScalingScheduleJob extends QuartzJobBean {
+	private Log logger = LogFactory.getLog(this.getClass());
+
+	@Override
+	protected void executeInternal(JobExecutionContext context) throws JobExecutionException {
+
+		SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(this);
+		logger.info("==========================================");
```

Comment:
I am confused. Are you going to use the slf4j on top of log4j? Why not just use log4j at that point. I can see if we are integrating existing code that uses some other logging framework.